server/src/main/java/de/hso/badenair/service/traveler/TravelerRepository.java

```diff
@@ -0,0 +1,8 @@
+package de.hso.badenair.service.traveler;
+
+import de.hso.badenair.domain.booking.Traveler;
+import org.springframework.data.repository.CrudRepository;
+
```

Comment:
```suggestion
@Repository
```
`@Repository` fehlt hier noch

Reply:
Hab ich das am richtigen Ort hinzugefügt? Bzw. stimmt das?
...
import org.springframework.stereotype.Repository;
import org.springframework.stereotype.Service;

import java.util.Optional;

@Repository
@Service
@RequiredArgsConstructor
public class TravelerService { ...